src/assets/workflows.json

```diff
@@ -118,6 +118,22 @@
                                         "selected": false
                                     }
                                 ]
+                            },
+                            {
+                                "label": "Trace Fire Perimeter",
+                                "name": "traceFireHydroPerimeter",
+                                "type": "subscription",
+                                "validators": 
+                                {
+                                    "required": true
+                                },
+                                "options": 
+                                [
+                                    {
+                                        "text": "",
```

Comment:
Can we add some text here? It looked a little funny in the workflow without a message. Maybe something like "Click "Next" to trace downstream" ?
![image](https://user-images.githubusercontent.com/40237491/182160415-94eb53e9-104b-401b-8bd9-a46da32f9f96.png)